components/page.jsx

```diff
@@ -10,8 +10,23 @@ var DevRibbon = (process.env.NODE_ENV === 'production' &&
                  process.env.SHOW_DEV_RIBBON !== 'on')
                 ? null
                 : require('./dev-ribbon.jsx');
+var config = require('../lib/config');
 
 var Page = React.createClass({
+  statics: {
+    handlerForPage: function(router, url) {
+      return router.match(url).routes[1].handler;
+    },
+    titleForHandler: function(handler) {
+      var title = 'Mozilla Learning';
+
+      if (handler.pageTitle) {
+        title = handler.pageTitle + ' - ' + title;
+      }
+
+      return title;
+    }
+  },
   contextTypes: {
     router: React.PropTypes.func
```

Comment:
You mean the `router`? I think that all `contextTypes` are required by default--at least it seems like React logs warnings if they're not present when the component is instantiated...